src/main/java/models/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package models;
+
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANGE = 100;
```

Comment:
better name suggestion
```suggestion
    private static final int MAX_BALL_NUMBER = 100;
```

Reply:
++